canonical-websites/www.ubuntu.com
# Remove docker ubuntu page

## Done

Remove `/containers/docker-ubuntu`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/containers/docker-ubuntu>
- See that you are redirected to `<http://0.0.0.0:8001/containers>`


## Issue / Card
Fixes #3284
